js/main.js

```diff
@@ -0,0 +1,55 @@
+let books = [];
+
+function showBook(book) {
+  let element;
+  const div = document.createElement('div');
+  element = document.createElement('p');
+  element.appendChild(document.createTextNode(book.title));
+  div.appendChild(element);
+  element = document.createElement('p');
+  element.appendChild(document.createTextNode(book.author));
+  div.appendChild(element);
+  element = document.createElement('button');
+  element.setAttribute('type', 'button');
+  element.appendChild(document.createTextNode('Remove'));
+  element.addEventListener('click', () => {
+    // remove the book from the screen
+    div.remove(book.id);
+    // remove the book from the local storage
+    const newArray = books.filter((book2) => book2.id !== book.id);
+    localStorage.setItem('books', JSON.stringify(newArray));
+  });
+  div.appendChild(element);
+  element = document.createElement('hr');
+  div.appendChild(element);
+  document.getElementById('book-cont').appendChild(div);
+}
+
+window.addEventListener('load', () => {
+  books = JSON.parse(localStorage.getItem('books'));
+  if (books === null) {
+    books = [];
+  }
+
+  books.forEach((element) => {
+    showBook(element);
+  });
+});
+
+document.getElementById('form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const randomId = Math.floor(Math.random() * 100);
+  const element = {
+    id: randomId,
+    title: document.getElementById('title').value,
+    author: document.getElementById('author').value,
+  };
+  books.push(element);
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20function%20to%20add%20a%20new%20book%20to%20the%20collection%2C%20with%20title%20and%20author.), Please a separate function to add a book to book collection and you can name it `addBook` then you can call it here inside the submit event listener.